src/modules/events.js

```diff
@@ -0,0 +1,202 @@
+    love.forEach((element) => {
+      element.addEventListener('click', (event) => {
+        const id = event.target.getAttribute('id');
+        loveCounter.countLove(event, id);
+        tvShowApi.sendLoveData(id);
+      });
+    });
+  }
+
+  // This function will handle the click event of comment button
+  handleCommentBtnClick = () => {
+    const commentBtn = document.querySelectorAll('.cmnt');
+    commentBtn.forEach((btn) => {
+      btn.addEventListener('click', (event) => {
+        this.showPopup(event);
+      });
+    });
+  };
+
+  // This function will create a new container for comment popUp and asign it into the mainElement
+  showPopup = (event) => {
+    const item = event.target.closest('.items');
+    const id = event.target.getAttribute('data-id');
+    const showImageSrc = item.querySelector('img').getAttribute('src');
+    const name = item.querySelector('h2').textContent;
+    const type = event.target.getAttribute('showType');
+    const language = event.target.getAttribute('lang');
+    const premiered = event.target.getAttribute('premiered');
+    const duration = event.target.getAttribute('runtime');
+    const commentPopup = `
+        <section class="popupContainer">
+            <div class="popupSection">
+                <div class="imgContainer">
+                    <div class="imgComment">
+                    <img id="showImg" src="${showImageSrc}" alt="">
+                    <img id="cancelImg" src="${cancel}" alt="">
+                    </div>
+                </div>
+                <div class="name">
+                    <h2>${name}</h2>
+                </div>
+                <div class="detailBlock">
+                    <ul class="leftBlock">
+                        <li>Type: ${type}</li>
+                        <li>Language: ${language}</li>
+                    </ul>
+                    <ul class="rightBlock">
+                        <li>Duration: ${duration} minutes</li>
+                        <li>Premiered: ${premiered}</li>
+                    </ul>
+                </div>
+                <div class="commentSection">
+                    <div class="title_count">
+                        <div class="title">Comments</div>
+                        <div class="count">(<span class="cmntCount"></span>)</div>
+                    </div>
+                    <ul class="comments">
+                    </ul>
+                </div>
+                <div class="commentsAddingSection">
+                    <div class="title">
+                        <h5>Add a comment</h5>
+                    </div>
+                    <form class="commentForm">
+                        <input placeholder="Your name" type="text" name="" id="name">
+                        <textarea placeholder="Your insights" name="" id="comment" cols="30" rows="10"></textarea>
+                        <button id="submit" type="submit">Comment</button>
+                    </form>
```

Comment:
- [ Optional ] I think it would be better if you clear the contents of the `input` and `textarea` fields once a user has submitted a comment successfully to the API, this will improve the User Experience should the user want to add another comment. After submitting the comment my details were still in the input fields, please see the screenshot below for a picture of what I'm talking about.

## Screenshot
![Screenshot from 2023-06-03 04-59-09](https://github.com/ibugithub/JavaScript_Group_Capstone/assets/16392046/6db82227-59a0-44a6-9faa-1362616dc7b1)

Reply:
Ok, now the fields will be cleared after each submit.